frame/staking/src/lib.rs

```diff
@@ -745,17 +745,46 @@ enum Releases {
 	V4_0_0,
 	V5_0_0, // blockable validators.
 	V6_0_0, // removal of all storage associated with offchain phragmen.
+	V7_0_0, // keep track of number of nominators / validators in map
 }
 
 impl Default for Releases {
 	fn default() -> Self {
-		Releases::V6_0_0
+		Releases::V7_0_0
 	}
 }
 
 pub mod migrations {
 	use super::*;
 
+	pub mod v7 {
+		use super::*;
+
+		pub fn pre_migrate<T: Config>() -> Result<(), &'static str> {
+			assert!(CurrentValidatorsCount::<T>::get().is_zero(), "CurrentValidatorsCount already set.");
+			assert!(CurrentNominatorsCount::<T>::get().is_zero(), "CurrentNominatorsCount already set.");
+			assert!(StorageVersion::<T>::get() == Releases::V6_0_0);
+			Ok(())
+		}
+
+		pub fn migrate<T: Config>() -> Weight {
+			log!(info, "Migrating staking to Releases::V7_0_0");
+			let validator_count = Validators::<T>::iter().count() as u32;
+			let nominator_count = Nominators::<T>::iter().count() as u32;
+
+			CurrentValidatorsCount::<T>::put(validator_count);
+			CurrentNominatorsCount::<T>::put(nominator_count);
+
+			StorageVersion::<T>::put(Releases::V7_0_0);
+			log!(info, "Done.");
```

Comment:
```suggestion
			log!(info, "Completed staking migration to Releases::V7_0_0");
```
Will make logging for `try-runtime on-runtime-upgrade` ever so slightly friendlier